api/search.go

```diff
@@ -3,12 +3,15 @@ package main
 import (
 	"log"
 	"net/http"
+
+	"google.golang.org/api/youtube/v3"
 )
 
 // Res struct for response
 type Res struct {
```

Comment:
The Queue should also be of this type not just url string, because we need to store all the meta data of the songs currently in queue and also display them on the frontend.

Reply:
@csoni111 I think we need to rework on the structure of the app, the current state is good enough to merge. Let's proceed?